cypress/integration/TokenGovernancePage/TokenGovernancePage.ts

```diff
@@ -0,0 +1,35 @@
+import {Given, Then} from "cypress-cucumber-preprocessor/steps";
+import {TokenGovernancePage} from "../../pageObjects/pages/TokenGovernancePage";
+import {LandingPage} from "../../pageObjects/pages/LandingPage";
+
+Given(`User has opened the {string} page`, (page) => {
+  LandingPage.openPage(page)
+});
+
+Then("User switches network for {string} and validates data", (network) => {
+  TokenGovernancePage.switchLNetworkAndValidateTokens(network)
+})
+
+Then("User filters super tokens by name for {string}", (network) => {
+  TokenGovernancePage.filterByTokenName(network)
+  TokenGovernancePage.resetFilter()
+})
+
+Then("User filters super tokens by symbol for {string}", (network) => {
+  TokenGovernancePage.filterByTokenSymbol(network)
```

Comment:
The input is not "network" but "tokenSymbol" :wink:

Reply:
Renamed to filterByTestTokenSymbolOn to make it more clear